libc/src/__support/File/dir.h

```diff
@@ -69,7 +71,7 @@ class Dir {
   // cleaned up.
   int close();
```

Comment:
all of the functions in this header should be marked `LIBC_INLINE`

Reply:
Just to confirm: including declarations with separate definitions in another TU?